core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -1,22 +1,24 @@
 package caliban.schema
 
 import java.util.UUID
-import scala.annotation.implicitNotFound
-import scala.concurrent.Future
-import scala.language.experimental.macros
+
 import caliban.CalibanError.ExecutionError
 import caliban.ResponseValue._
 import caliban.Value._
 import caliban.introspection.adt._
 import caliban.parsing.adt.Directive
-import caliban.schema.Annotations.{ GQLDeprecated, GQLDescription, GQLDirective, GQLInputName, GQLInterface, GQLName }
+import caliban.schema.Annotations._
 import caliban.schema.Step._
 import caliban.schema.Types._
 import caliban.{ InputValue, ResponseValue }
 import magnolia._
-import zio.{ URIO, ZIO }
-import zio.stream.ZStream
 import zio.query.ZQuery
+import zio.stream.ZStream
+import zio.{ URIO, ZIO }
+
+import scala.annotation.implicitNotFound
```

Comment:
I used to have java imports, then scala imports, then the rest but maybe better to stick with IntelliJ defaults which is what you did.